### PR TITLE
New chair for mentoring

### PR DESCRIPTION
**Pull request by rgaiacs**
@ChristinaLK Could you send me a email or pull request filling the blanks on the last paragraph? Thanks.

### CODE REVIEW

File: _posts/2016/03/2016-03-30-changes-on-mentoring.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+The first quarter of 2016 is almost over and so far the Mentoring Subcommittee
+have been very quiet although we continue hosting the post workshop debriefing
+sessions and the pre-workshop helping session.
+
+We have been quiet because we were busy with the Steering Committee election
+and reorganization of the volunteer hours from our members since six of them
+(Belinda Weaver, Bill Mills, Karin Lagesen, Kate Hertweck, Rayna M Harris and I)
+were elected as Steering Committee members.
+In addition to this some members had to temporary reduce their volunteer hours
+for various reasons but are now backing to help us.
```
Comment (wking) [inline _posts/2016/03/2016-03-30-changes-on-mentoring.md]: Maybe “backing to help us” → “back to their usual level”?

File: _posts/2016/03/2016-03-30-changes-on-mentoring.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+In the upcoming weeks you will hear more news from the Mentoring Subcommittee
+but for now I want to ask you to welcome Christina Koch as the new chair of the
+Subcommittee. Christina will replace me on this position and I can say that the
+Subcommittee couldn't be in better hands. For those that don't know Christina,
+she is one of our instructors working at the University of Wisconsin-Madison,
+last year she was one of the maintainers of our Shell lesson
+and are now part of our Instructor Trainer team.
```
Comment (wking) [inline _posts/2016/03/2016-03-30-changes-on-mentoring.md]: “are” → “is”.

**Comment by rgaiacs** [comment]
@ChristinaLK Do you have time for this before Friday?

**Comment by ChristinaLK** [comment]
Yes - I wanted to get through today's meetings before adding anything.

**Comment by ChristinaLK** [comment]
Hi @rgaiacs sorry this took so long.  I submitted changes as a PR against your repo/branch.

**Comment by rgaiacs** [comment]
Close by 7f140fa.

**Pull request closed without merging**